linux/deploy_vm/ubuntu/reconfigure_ubuntu_iso_vm.yml

```diff
@@ -18,6 +18,12 @@
     dest: ubuntu_install_method.yml
     mode: '0644'
 
+- name: Read ubuntu install method file
+  command: cat ubuntu_install_method.yml
+  register: file_result_ubuntu_install_method
+
+- debug: var=file_result_ubuntu_install_method
```

Comment:
debug also need a 'name'.

Reply:
Done

Reply:
@Tomorrow9 add the screenshots of test result